docs/UserGuide.adoc

```diff
@@ -62,11 +62,17 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
 
 [TIP]
 A person can have any number of tags (including 0)
+A person can be added with just the name being specified
+Phone number, email and address are optional during the addition of a person
+The Order in which name, phone number, email, address and tag are specified is non-specific
```

Comment:
Repetitive information with line 65 and 66.
Also needs clearer phrasing for line 67 ("order does not matter", perhaps?)

Reply:
ok noted